examples/webhook-authz/README.md

```diff
@@ -0,0 +1,106 @@
+# Example Webhook authorizer
+
+This directory contains the scaffolding necessary to use a webhook authorizer
+with the standalone cluster registry. It is not meant to be used with the
+aggregated cluster registry.
+
+NOTE: this example is not meant for production use! Do not deploy this into a
+production environment.
+
+## Usage
+
+1.  Deploy a standalone cluster registry. You can use
+    [crinit](https://github.com/kubernetes/cluster-registry/blob/master/docs/userguide.md#standalone).
+
+2.  Create a self-signed SSL cert for the cluster registry webhook authorizer
+    (or skip this step and use an existing SSL cert).
+
+```sh
+openssl req -x509 -newkey rsa:4096 -keyout key.pem -out cert.pem -nodes
+```
+
+3.  Create a ConfigMap that contains the files needed for the cluster registry
+    webhook authorizer. If you deployed your cluster registry into a namespace
+    other than `clusterregistry`, change the namespace below accordingly.
+
+```sh
+kubectl create configmap bitesize -n clusterregistry \
+  --from-file=./rules.hcl --from-file=./webhook-config.yaml \
+  --from-file=./key.pem --from-file=./cert.pem
+```
+
+4.  Create the webhook authorizer. Note that you may need to change the
+    namespace if you did not deploy the cluster registry into the
+    `clusterregistry` namespace.
+
+```sh
+kubectl apply -f ./authz.yaml -n clusterregistry
+```
+
+5.  Update the cluster registry deployment as follows.
+
+```yaml
+...
+      - command:
+        - ./clusterregistry
+        - --advertise-address=<...>
+        - --bind-address=0.0.0.0
+        - --client-ca-file=/etc/clusterregistry/apiserver/ca.crt
+        - --etcd-servers=http://localhost:2379
+        - --secure-port=8443
+        - --tls-cert-file=/etc/clusterregistry/apiserver/server.crt
+        - --tls-private-key-file=/etc/clusterregistry/apiserver/server.key
+        - --token-auth-file=/etc/clusterregistry/apiserver/token.csv
```

Comment:
Did you launch with `--apiserver-enable-token-auth`?

Reply:
I did. I've modified this section of the `README` to only include things that need to be added, hopefully to make it less confusing.